packages/@tailwindcss-node/src/index.ts

```diff
@@ -6,5 +6,5 @@ export * from './compile'
 // not necessary.
 if (!process.versions.bun) {
   let localRequire = Module.createRequire(import.meta.url)
-  Module.register(pathToFileURL(localRequire.resolve('@tailwindcss/node/esm-cache-loader')))
+  Module.register?.(pathToFileURL(localRequire.resolve('@tailwindcss/node/esm-cache-loader')))
```

Comment:
Maybe add a comment as to _why_ we're doing the conditional check?

Something like this maybe?

```
// Register was not added until Node v18.19.0 and v20.6.0
// Not calling it means that while ESM dependencies don't get reloaded the actual included files will because they cache bust directly by using `?id=…`
```

Reply:
(but appropriately wrapping the comment ofc)

Reply:
Yep, for sure!